### ISSUE TITLE
NTR: Muscle biopsy: citrate synthase activity high (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9079]
New term request Muscle biopsy: citrate synthase activity high (segal_050203183832):

**Comment by pnrobinson** [comment]
@MickeySegal
note using "official" name of enyzme

Adding new term: 
- ID: HP:6000195
- label: Elevated mitochondrial citrate synthase synthase activity
- parents: [Abnormal enzyme activity in muscle tissue (HP:0034526)](https://hpo.jax.org/app/browse/term/HP:0034526)
- pmid: PMID:25057335
- def: Activity of the enzyme mitochondrial citrate synthase (EC:2.3.3.1) in muscle tissue is above the upper limit of normal.
- comment: Citrate synthase (CS) activity is a validated biomarker for mitochondrial density in skeletal muscle. CS activity is also used as a biochemical marker of the skeletal muscle oxidative adaptation to a training intervention, and a relationship between changes in whole body aerobic capacity and changes in CS activity is often assumed.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9079
- synonyms:

**Comment by MickeySegal** [comment]
Do you mean to have a repeated word in "mitochondrial citrate synthase synthase"?
